addons/l10n_ch/wizard/setup_wizards.py

```diff
@@ -18,4 +18,4 @@ def _onchange_recompute_qr_iban(self):
 
     @api.depends('partner_id', 'company_id')
     def _compute_l10n_ch_display_qr_bank_options(self):
-        self.res_partner_bank_id._compute_l10n_ch_display_qr_bank_options(self)
+        self.l10n_ch_display_qr_bank_options = self.res_partner_bank_id.l10n_ch_display_qr_bank_options
```

Comment:
While we're at it, could we take the opportunity to iterate over `for wizard in self `please ? 
Otherwise, LGTM ; maybe @oco-odoo wants to be kept in the loop :)

Reply:
Requested change is done please review it!

Reply:
Review is done. I don't have r+ rights.